### PR TITLE
Send event in correct place + removePersistentMenu

### PR DESCRIPTION
**Pull request by etiennea**

### CODE REVIEW

File: lib/Botly.js [hunk 1]
```diff
@@ -244,6 +261,7 @@ Botly.prototype.upload = function (options, callback) {
 };
 
 Botly.prototype.send = function (options, callback) {
+    let that = this;
```
Comment (miki2826) [inline lib/Botly.js]: why do you need this line?
Arrow function keeps the scope and the `this` should be the correct one

Comment (etiennea) [inline lib/Botly.js]: Indeed! Well spotted, I have amended my pull request.

**Comment by etiennea** [comment]
Humm makes the tests fail

File: .travis.yml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 language: node_js
 node_js:
-   - "4.4.3"
+   - "8.1.4"
```
Comment (miki2826) [inline .travis.yml]: Why update node? I check it with the minimum version required

**Pull request merged**